tests/README.md

```diff
@@ -44,6 +44,29 @@ tests
 
 ## Contibute to terraform rhcs provider tests
 Please read the structure and contribte code to the correct place
+### Contribute to day1 case
+* Define manifests files under terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters
+* Define profile in terraform-provider-rhcs/tests/ci/profiles/tf_cluster_profile.yml
+* If any other data can be created by terrafom provder, define steps under terraform-provider-rhcs/tests/tf-manifests/<provider>/<resource name>
+* Define prepare steps according to the profile in terraform-provider-rhcs/tests/ci/profile_handler.go
+#### Day1 cases only do creation step, we have to define a day1 case to verify the day1 works well
```

Comment:
What does it mean "define a day1 case to verify the day1 works well"?
If it is done after Day1 (cluster creation) - how is it different from Day2 cases?

Reply:
@amalykhi The day1 creation only does creation, it won't check if the configuration succeed. For example, when you create a cluster with admin enabled in day1, the cluster will create with admin enabled. But actually, we need to verify that the cluster is indeed created with admin enabled which means we have to login to the cluster with the admin username/password to make sure the configuration/creation worked. 
Why we mark it as day1-post rather than day2, because OCM-backend meets the problem now that some day2 cases may remove/change the day1 configuration no matter caused by not good coding or unrecoverable step and cases run randomly. I don't want TF provider automation meets same issue. So we label them differently then in future we can run them by order day1-post first, and then day2 and then maybe day3(TBD, those cases will cause destructive results)